### DISCUSSION TITLE
Some types become private in v6.rc3?

### DISCUSSION THREAD

**Discussion by michael-land** [react-hook-form/react-hook-form#1835] [Ideas] [answered]
![image](https://user-images.githubusercontent.com/33362998/84276577-19d19b80-aaf8-11ea-8156-0cb86e18df14.png)

All these types are quite useful

**Comment by bluebill1049** [comment, marked as answer]
We want to limit the number of types to be exposed. let's discuss whether those are essential or user can easily re-create them at their source code. 

cc: @kotarella1110

**Reply by bluebill1049** [reply]
it's also important to aware, we were taking easy on Types in terms of breaking change, which we shouldn't, from v6 we properly document them down and be extra careful on what types that we going to expose.
